Documentation/UsingQtCreator.md

```diff
@@ -0,0 +1,20 @@
+## Using Qt Creator for working with SerenityOS
+
+You can follow the video tutorial [on YouTube](https://www.youtube.com/watch?v=NcTEaRB_cak), or the instructions below.
+
+First, make sure you have a working toolchain and can build and run SerenityOS. Go [here](https://github.com/SerenityOS/serenity/blob/master/Documentation/BuildInstructions.md) for instructions for setting that up.
+
+* Install [Qt Creator](https://www.qt.io/offline-installers). You don't need the entire Qt setup, just click 'Qt Creator' on the left side, and install that.
+* Open Qt Creator, select `File -> New File or Project...`
+* Select `Import Existing Project`
+* Give it a name, and navigate to the root of your SerenityOS project checkout. Click Next.
+* Wait for the file list to generate. This can take a minute or two!
+* Ignore the file list, we will overwrite it later. Click Next.
+* Set `Add to version control` to `<None>`. Click Finish.
+* In your shell, go to your SerenityOS project directory, and invoke the `Meta/refresh-serenity-qtcreator.sh` script to regenerate the `serenity.files` file. You will also have to do this every time you delete or add a new file to the project.
+* Edit the `serenity.config` file (In Qt Creator, hit ^K or CMD+K on a Mac to open the search dialog, type the name of the file and hit return to open it)
+* Add the following `#define`s to the file: `DEBUG`, `SANITIZE_PTRS`, `KERNEL`, `USERLAND`. Depending on what you are working on, you need to have one of those last two defines commented out. If you're planning on working in the userland, comment out `#define KERNEL`. If you're working on the Kernel, then comment out `#define USERLAND`.
```

Comment:
`USERLAND` is not used anywhere anymore. There is only `KERNEL`. No `KERNEL` means userspace.